appveyor.yml

```diff
@@ -1,26 +1,36 @@
+build: off
+cache:
+  - '%APPDATA%\npm-cache'
+clone_depth: 1
+skip_branch_with_pr: true
+skip_commits:
+  files:
```

Comment:
Wouldn't it be better to whitelist? Using `only_commits.files`. So only `.js` and `.json` files.

Reply:
Hmm, I don't know if that's new since https://github.com/avajs/ava/commit/127854ad549da9fbe98e8172883101ebea53fe8c#diff-180360612c6b8c4ed830919bbb4dd459. But then you'd want to run CI if you change say a fixture file or a snapshot.

Reply:
Good point.